README.md

```diff
@@ -178,6 +178,20 @@ You can then run the tests by executing the following command in your terminal:
 pytest tests
 ```
 
+## The default Neurobagel SPARQL query
+
+`docs/default_neurobagel_query.rq` contains a sample default SPARQL query sent by the Neurobagel API to a graph database to retrieve all available phenotypic and imaging data.
```

Comment:
```suggestion
`docs/default_neurobagel_query.rq` contains a sample default SPARQL query sent by the Neurobagel API to a graph database to retrieve all available phenotypic and imaging data.
This file is mainly intended for reference because in normal operations, 
the API will always talk to the graph on behalf of the user.
```